packages/framework/src/Model/Seo/SeoSettingFacade.php

```diff
@@ -45,7 +40,7 @@ public function getDescriptionMainPage($domainId)
      * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig[] $domainConfigs
      * @return string[]
      */
-    public function getDescriptionsMainPageIndexedByDomainIds(array $domainConfigs)
+    public function getDescriptionsMainPageIndexedByDomainIds(array $domainConfigs): array
```

Comment:
adding return type is a BC-break. please remove it.

Reply:
Removed.